content/microcontrollers/esp8266.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "ESP8266"
+weight: 3
+---
+
+The [Espressif ESP8266]() is a tiny development board based on the Xtensa [ESP8266]() family of SoC.
+
+## Interfaces
+
+| Interface | Hardware Supported | TinyGo Support |
+| --------- | ------------- | ----- |
+| GPIO      | YES | YES |
+| UART      | YES | YES |
+| SPI      | YES | Not Yet |
+| I2C      | YES | Not Yet |
+| ADC      | YES | Not Yet |
+| PWM      | YES | Not Yet |
+
+## Machine Package Docs
+
+[Documentation for the machine package for the ESP8266](../machine/esp8266)
+
+## Flashing
+
+### CLI Flashing on Linux
+
+You need to install the Espressif toolchain for Linux to use TinyGo with the ESP8266. Note that it is the same download as for the ESP32: 
```

Comment:
The ESP8266 normally has a different toolchain, but in the case of TinyGo the ESP8266 also uses the ESP32 toolchain.

So you could for example say:

```suggestion
You need to install the same toolchain for the ESP8266 as is used for the ESP32 to use the ESP8266 with TinyGo:: 
```

Reply:
OK changed.